### ISSUE TITLE
[Bug]  OSError: [WinError 87] The parameter is incorrect: '.'  on Windows with mlc_chat.rest nightly build

### ISSUE THREAD

**Issue by Sing-Li** [mlc-ai/mlc-llm#588]
## 🐛 Bug
On Windows 11, running in miniconda3 (Python 3.10.10)
After updating to latest nightly build, running `mlc_chat.rest` with `--help`  crashes.

## To Reproduce

1.  create a miniconda3 environment in Windows 11
2.  install latest nightly using `pip`
3.  run `python -m mlc_chat.rest --help`

```
python -m mlc_chat.rest --help
Traceback (most recent call last):
  File "C:\Users\allmy\miniconda3\lib\runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "C:\Users\allmy\miniconda3\lib\runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "C:\Users\allmy\miniconda3\lib\site-packages\mlc_chat\__init__.py", line 6, in <module>
    from .chat_module import ChatModule
  File "C:\Users\allmy\miniconda3\lib\site-packages\mlc_chat\chat_module.py", line 26, in <module>
    _LIB, _LIB_PATH = _load_mlc_llm_lib()
  File "C:\Users\allmy\miniconda3\lib\site-packages\mlc_chat\chat_module.py", line 18, in _load_mlc_llm_lib
    os.add_dll_directory(path)
  File "C:\Users\allmy\miniconda3\lib\os.py", line 1118, in add_dll_directory
    cookie = nt._add_dll_directory(path)
OSError: [WinError 87] The parameter is incorrect: '.'
```

## Expected behavior

`mlc_chat.rest` should print out help information;   as it does on the Mac with same nightly build

## Environment

 - Platform  AMD Vulkan
 - Operating system Windows 11
 - How you installed MLC-LLM (`conda`, source):  followed nightly build install instructions
 - How you installed TVM-Unity (`pip`, source):  no
 - Python version (e.g. 3.10): 3.10.10
 - GPU driver version (if applicable):
 - CUDA/cuDNN version (if applicable):
 - TVM Unity Hash Tag (`python -c "import tvm; print('\n'.join(f'{k}: {v}' for k, v in tvm.support.libinfo().items()))"`, applicable if you compile models):

## Additional context

Found zstd.dll missing earlier.  But `conda install zstd zlib` fixed it.

**Comment by yzh119** [comment]
Should have been resolved in #589

**Comment by Sing-Li** [comment]
Thanks.  Will confirm with next Nightly🙏

**Comment by Sing-Li** [comment]
confirmed fixed in nightly by #589  - closing.
